notebooks/llm-agent-functioncall/llm-agent-functioncall-qwen.ipynb

```diff
@@ -0,0 +1,868 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #123.    demo.run()
</pre><p>could you please add try except with share=True argument?</p><p> </p><p>in some environments, it is not possible to launch gradio without server sharing</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2156/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>